qa/rpc-tests/wallet.py

```diff
@@ -6,6 +6,27 @@
 from test_framework.test_framework import BitcoinTestFramework
 from test_framework.util import *
 
+def check_array_result(object_array, to_match, expected):
```

Comment:
Indeed.. It is used by multiple tests. I think it comes in handy for current and future test cases. If you guys agree I guess this could be moved to `test_framework/util.py` to be reused whenever needed..